tests/pytests/functional/states/test_npm.py

```diff
@@ -1,9 +1,13 @@
 import pytest
+from salt.exceptions import CommandExecutionError
 
 
 @pytest.fixture(scope="module", autouse=True)
 def install_npm(sminion):
-    sminion.functions.pkg.install("npm")
+    try:
+        sminion.functions.pkg.install("npm")
```

Comment:
if we're going to install it, should we use pkg.installed so it won't upgrade if it's already installed?  I think that would greatly reduce the flakiness and still give us more of a chance that this doesn't accidentally get skipped.